main.css

```diff
@@ -2,7 +2,5 @@
 
 @import url('css/feedback.css');
  
-@import url('css/help_dialog.css');
```

Comment:
Actually it turns out we don't use this file at all (cf `#devops` in slack), so I updated this to reflect the merged state of develop, which has no main.css -- thanks for following-up!